metrics-runtime/src/common.rs

```diff
@@ -23,6 +25,17 @@ pub enum Scope {
 }
 
 impl Scope {
+    /// Adds a new part to this scope.
+    pub fn add_part(&mut self, part: ScopedString) {
```

Comment:
I should probably switch `Sink` to use this new method instead of manually destructing the scope and adding things manually.

Reply:
All set!